src/commands/utility/info/voice.js

```diff
@@ -0,0 +1,40 @@
+const { Command, SwitchbladeEmbed, CommandError } = require('../../../')
+const moment = require('moment')
+
+module.exports = class InfoVoice extends Command {
+  constructor (client) {
+    super({
+      name: 'voice',
+      parent: 'info',
+      requirements: { guildOnly: true },
+      parameters: [{
+        type: 'string',
```

Comment:
Will be better if you use channel param and customize it to support voice channels.